packages/common/src/hardhat/tasks/artifacts.ts

```diff
@@ -213,7 +213,7 @@ export type { ${normalizeClassName(contractName)}Web3Events };\n`
 
     // Constructs a mapping of name to address function for nodejs.
     fs.appendFileSync(out, "const addressFunctions = {\n");
-    Object.keys(addresses).forEach((name) => fs.appendFileSync(out, `  ${name}: get${name}Address,\n`));
+    Object.keys(addresses).forEach((name) => fs.appendFileSync(out, `  ${name}: get${name}StaticAddress,\n`));
```

Comment:
is this to match the fe api?

Reply:
We had internal functions called the same thing that are used for something else. So I renamed those functions to make room for the new ones.

These functions differ in that they map only the static addresses, not the dynamic ones (i.e. hardhat testing addresses).